test/integration_testcase.go

```diff
@@ -118,7 +140,14 @@ func getFile(path string) (bool, string, error) {
 	if err != nil {
 		return false, "", nil
 	}
-	f, err := ioutil.ReadFile(path)
+
+	// Ensure it's the absolute, symlink-less path we're returning
```

Comment:
So I removed this from `Path` in `test.go` [here](https://github.com/golang/dep/blob/master/test/test.go#L509)  as in my `final` I didnt have a `lock.json` (is some discussion above).

Will remove it from here as well for the moment.